### ISSUE TITLE
It works

### ISSUE THREAD

**Issue by pblanchardie** [AnirudhKaranth/Acadamix#1]
![image](https://github.com/AnirudhKaranth/Acadamix/assets/676407/97636de1-d102-4ffa-a20d-0c7dbf5e2bf0)

**Comment by AnirudhKaranth** [comment]
![image](https://github.com/AnirudhKaranth/Acadamix/assets/97871145/6a13174a-c868-44d8-8e0a-d64184f842f7)

**Comment by AnirudhKaranth** [comment]
![image](https://github.com/AnirudhKaranth/Acadamix/assets/97871145/ad8b8e1f-c733-48cf-bfca-01aeedbfbed2)

**Comment by pblanchardie** [comment]
Btw, could you mark my answer as accepted on stackoverflow please?

**Comment by AnirudhKaranth** [comment]
Yep.

**Comment by AnirudhKaranth** [comment]
@pblanchardie  Could you please explain about the issue in my testing configuration

**Comment by pblanchardie** [comment]
Yes, your tests are not configured and break the build. You could copy and paste your application.properties in src/test/resources so your tests are configured with the same db as in main.

Or, for now, you could delete the test in src/test/java.

Or simply run mvn compile (not package) so it compiles withojt running tests.

Or run the full build (mvn package or others) with skipTests.

Try and I'll submit a PR if needed.

**Comment by AnirudhKaranth** [comment]
There is no such folder as src/test/resources, I have src/test/java.
I copied and pasted application.properties in it. But still getting same error

**Comment by pblanchardie** [comment]
Please copy and paste the full output of the mvn compile command. It should not run tests.


You have to create the directory src/test/resources and put application.properties in it to configure the testing environment.
Then check the result of the maven build.
But I suggest that for now you don't waste time with it.

**Comment by AnirudhKaranth** [comment]
![image](https://github.com/AnirudhKaranth/Acadamix/assets/97871145/580245e1-879b-4b19-9417-e50e4a755e7c)

**Comment by AnirudhKaranth** [comment]
@pblanchardie  Even though the it compiled successfully and generated-sources folder is created. My annotations folder is empty.

![image](https://github.com/AnirudhKaranth/Acadamix/assets/97871145/2d30bded-0db6-41a7-aaf5-a6f76e69ded7)

**Comment by AnirudhKaranth** [comment]
@pblanchardie Hey, Its Working.  **Thank you so much**.
